### PR TITLE
feature/1806-feedback-char-limit

### LINKED ISSUE #1806: Set Maximum Character Limit User is Allowed to Enter in Feedback Responses
**Environment:** https://staging.fightpandemics.work/

**Steps to reproduce:**

- Click the feedback icon on the top right of the homepage when user is not signed in.

- Answer question on screen 1. When user reaches screen 2, enter as many characters in the response fields as the user wants. When user reaches screen 3, enter as many numbers in the age response field as the user wants. Submit feedback.

**Actual results:**

When user is on screen 2, he/she can enter as many characters in the response fields with no character limit.
When user is on screen 3, he/she can enter as many numbers in the response field with no integer limit.
Feedback can be submitted successfully.

**Expected Results:**

A max character limit should be set for the response fields on screen 2 (e.g. 1000 character limit).
A max 3 number limit (assuming there are people over age 99) should be set for the age response field on screen 3.

image

image

### PR DESCRIPTION
**Pull request by rz-0605**
### What does this pull request aim to achieve? 💯
This PR is a duplicate of #1871  (original author is inactive) and address some issues and features mentioned in the comments.
_Please be concise and link any related issue(s):_ fixes issue #1806 

- [x] I have checked that no one else is working on similar changes.
- [x] I have read the latest [**README**](README.md) and understand the development workflow.
- [x] The name of this branch is: **`feature/1806-feedback-char-limit`**, or **`hotfix/<branch_name>`** if this is a hotfix, where `<branch_name>` briefly describes the issue(s) resolved and is prefixed with the issue number(s) (e.g. `1127-1130-update-git-branching-model`). The branch is created in a cloned version of this repo, **not a forked repo**.
- [x] This branch is rebased/merged with the **latest staging** (or the **latest production** if it is a hotfix).
- [x] There are no merge conflicts.
- [x] There are no console warnings and errors.
- [x] On the right hand side of this PR, I have linked the appropriate issue(s) under "Issues" and added a project progress under "Projects".
- [x] The title describes the issue(s) being addressed using format: **`<issue_numbers> - <brief_issue_description>`** (e.g. `#1127/#1130 - Update Git Branching Model`)

### CODE REVIEW

File: client/src/components/Input/InputInfo.js [hunk 1]
```diff
@@ -0,0 +1,11 @@
+import styled from 'styled-components';
+
+import InputError from "./InputError";
+
+import { theme } from "constants/theme";
+
+const InputInfo = styled(InputError)`
+    color: ${props => props.error ? theme.colors.red : theme.colors.lightGray}
```
Comment (Zaydme) [inline client/src/components/Input/InputInfo.js]: Can you please use a darker grey than this, it is barely visible
image
other than that, it looks good to me!

**Review by Zaydme — APPROVED** [review]
LGTM! Thanks @rz-0605 @speed-o-soundSonic

**Review by rabbi08 — APPROVED** [review]
Verified the fix on the feature branch as working fine: PASS
Unable to enter 'value<1' and 'value >150', valid message is displayed to enter valid age. 
Please see below GIF and screenshot:

ezgif com-video-to-gif

image

**Review by ddveloper — APPROVED** [review]
LGTM! Thanks @rz-0605 @speed-o-soundSonic

**Pull request merged**